.github/workflows/linux.yml

```diff
@@ -31,15 +31,25 @@ jobs:
             libgit/target
           key: ${{ runner.os }}-cargo-${{ matrix.version }}
 
+      - name: Install virtualenv
+        run: pip install virtualenv
+
       - name: Install ${{ matrix.version }}
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.version }}-x86_64-unknown-linux-gnu
           profile: minimal
           override: true
 
-      - name: check build
-        run: cd libgit && cargo check --all --bins --examples --tests
+      - name: Create virtualenv
+        run: virtualenv venv 
+
+      - name: Install dependencies
+        run: make env
 
-      - name: tests
-        run: cd libgit && cargo test --all --all-features --no-fail-fast
+      - name: Run tests
+        run: make test
+        env:
+          DYNACONF_GITEA__HOST : "https://git.batsense.net"
```

Comment:
Similar question to the one posed earlier.

Reply:
answered above https://github.com/forgefedv2/interface/pull/17#discussion_r756642848.